radar-auth/build.gradle

```diff
@@ -8,9 +8,11 @@ plugins {
 
 sourceCompatibility = 1.8
 
-ext.jacksonVersion='2.9.3'
+ext {
+    jacksonVersion = '2.9.3'
```

Comment:
Would it be safe update this to the latest version?  '2.9.7'

Reply:
Yes. Looking at spring, it still demands Jackson `2.8.10`, so that is the version used in the main project. But for `radar-auth` we can use the latest.